ocs_ci/ocs/platform_nodes.py

```diff
@@ -818,6 +824,271 @@ def _prepare_node(self, node_id):
             except Exception:
                 logger.error("Failed to create RHEL node")
                 raise
+        elif self.node_type == 'RHCOS':
+            conf = self._prepare_rhcos_node_conf()
+            conf['node_id'] = node_id
+            try:
+                self.aws_instance_obj = self._prepare_upi_rhcos_node(conf)
+            except Exception:
+                logger.error("Failed to create RHCOS node")
+                raise
+            self.approve_pending_csr()
+
+    def _prepare_rhcos_node_conf(self):
+        """
+        Merge default RHCOS node configuration for rhcos node
+        along with the user provided config
+
+        Returns:
+            conf (dict): A dictionary of merged user and default values
```

Comment:
Change to:

dict: A dictionary of merged user and default values

Reply:
Done